README.md

```diff
@@ -303,6 +303,16 @@ To enable inner join add this to your configuration.
 ```
 
 
+There is also an ORM Query Type for LeftJoin.  This join type is commonly used to fetch an empty right side of a relationship.
+
+Left Join is not included by default in the `zf-doctrine-querybuilder.global.php.dist`.
```

Comment:
For the same reason innerJoin is not included in the .dist.  The join filters allow deeper querying of data.  Not including them is erring on the side of caution.  Examples are all theoretical.  Including the ability to join tables IMHO should be left to the implementing developer.

For instance given a resource which can return users you could inner join the user table to the user acl table and fetch all users who are just administrators.